statescript/executor.go

```diff
@@ -243,9 +243,9 @@ func execute(name string, timeout time.Duration) error {
 
 	if len(bts) > 0 {
 		if len(bts) > 10*1024 {
-			log.Errorf("stderr collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
+			log.Errorf("The standard-error collected while running script %s [%s] (Truncated to 10KB)", name, bts[:10*1024])
```

Comment:
Usage of "The" here is weird, I think the original was better. But maybe a compromise? `Stderr messages collected...`

Reply:
How about: `Collected stderr while running...` ?

Reply:
:+1: